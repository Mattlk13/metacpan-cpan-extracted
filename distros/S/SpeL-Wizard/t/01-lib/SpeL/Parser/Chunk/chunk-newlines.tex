\begin{align}
  (a) && z=7-3\jz && \Ree{z} = 7 && \Ima{z} = -3
  \\
  (b) && z=-\sqrt{2}-\frac{1}{2} \jz && \Ree{z} = -\sqrt{2} && \Ima{z} =
  -\frac{1}{2}
  \\
  (c) && z=-\pi \jz && \Ree{z} = 0 && \Ima{z} = - \pi
\end{align}
