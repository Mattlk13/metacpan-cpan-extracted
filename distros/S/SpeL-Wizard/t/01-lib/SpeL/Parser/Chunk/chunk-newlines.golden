$tex = bless( {
  'path' => 't/01-lib/SpeL/Parser/Chunk',
  'tree' => bless( {
    'ElementList' => bless( {
      'Element' => [
        bless( {
          'MathUnit' => [
            bless( {
              '' => '(a) && z=7-3\\jz && \\Ree{z} = 7 && \\Ima{z} = -3
  ',
              'Left' => bless( {
                'Component' => bless( {
                  '' => 'a',
                  'Left' => bless( {
                    'Component' => bless( {
                      'Alphabet' => 'a',
                      'trailingws' => ''
                    }, 'SpeL::Object::Variable' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                }, 'SpeL::Object::Relation' ),
                'Rest' => bless( {
                  'ws' => ''
                }, 'SpeL::Object::Expressionrest' ),
                'cbracket' => ')',
                'obracket' => '('
              }, 'SpeL::Object::Expression' ),
              'RelationChain' => [
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Alphabet' => 'z',
                      'trailingws' => ''
                    }, 'SpeL::Object::Variable' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Realnumber' => bless( {
                        'Sign' => '',
                        'Value' => '7',
                        'trailingws' => ''
                      }, 'SpeL::Object::Realnumber' )
                    }, 'SpeL::Object::Number' ),
                    'Rest' => bless( {
                      'Op' => bless( {
                        'Op' => '-'
                      }, 'SpeL::Object::Binop' ),
                      'Remainder' => bless( {
                        'Component' => bless( {
                          'Realnumber' => bless( {
                            'Sign' => '',
                            'Value' => '3',
                            'trailingws' => ''
                          }, 'SpeL::Object::Realnumber' )
                        }, 'SpeL::Object::Number' ),
                        'Rest' => bless( {
                          'Op' => bless( {
                            'Op' => ''
                          }, 'SpeL::Object::Binop' ),
                          'Remainder' => bless( {
                            'Component' => bless( {
                              'Name' => 'jz',
                              'trailingws' => ' '
                            }, 'SpeL::Object::Command' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Expressionrest' )
                      }, 'SpeL::Object::Expression' )
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Args' => [
                        bless( {
                          '' => 'z',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'z',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' )
                      ],
                      'Name' => 'Ree',
                      'trailingws' => ' '
                    }, 'SpeL::Object::Command' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Realnumber' => bless( {
                        'Sign' => '',
                        'Value' => '7',
                        'trailingws' => ' '
                      }, 'SpeL::Object::Realnumber' )
                    }, 'SpeL::Object::Number' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Args' => [
                        bless( {
                          '' => 'z',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'z',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' )
                      ],
                      'Name' => 'Ima',
                      'trailingws' => ' '
                    }, 'SpeL::Object::Command' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Realnumber' => bless( {
                        'Sign' => '-',
                        'Value' => '3',
                        'trailingws' => '
  '
                      }, 'SpeL::Object::Realnumber' )
                    }, 'SpeL::Object::Number' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                }
              ]
            }, 'SpeL::Object::Relation' ),
            bless( {
              '' => '(b) && z=-\\sqrt{2}-\\frac{1}{2} \\jz && \\Ree{z} = -\\sqrt{2} && \\Ima{z} =
  -\\frac{1}{2}
  ',
              'Left' => bless( {
                'Component' => bless( {
                  '' => 'b',
                  'Left' => bless( {
                    'Component' => bless( {
                      'Alphabet' => 'b',
                      'trailingws' => ''
                    }, 'SpeL::Object::Variable' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                }, 'SpeL::Object::Relation' ),
                'Rest' => bless( {
                  'ws' => ''
                }, 'SpeL::Object::Expressionrest' ),
                'cbracket' => ')',
                'obracket' => '('
              }, 'SpeL::Object::Expression' ),
              'RelationChain' => [
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Alphabet' => 'z',
                      'trailingws' => ''
                    }, 'SpeL::Object::Variable' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Sqrt' => bless( {
                        'Argument' => bless( {
                          'Component' => bless( {
                            'Realnumber' => bless( {
                              'Sign' => '',
                              'Value' => '2',
                              'trailingws' => ''
                            }, 'SpeL::Object::Realnumber' )
                          }, 'SpeL::Object::Number' ),
                          'Rest' => bless( {
                            'ws' => ''
                          }, 'SpeL::Object::Expressionrest' )
                        }, 'SpeL::Object::Expression' )
                      }, 'SpeL::Object::Squareroot' )
                    }, 'SpeL::Object::Function' ),
                    'Rest' => bless( {
                      'Op' => bless( {
                        'Op' => '-'
                      }, 'SpeL::Object::Binop' ),
                      'Remainder' => bless( {
                        'Component' => bless( {
                          'den' => bless( {
                            'Realnumber' => bless( {
                              'Sign' => '',
                              'Value' => '2',
                              'trailingws' => ''
                            }, 'SpeL::Object::Realnumber' )
                          }, 'SpeL::Object::Number' ),
                          'num' => bless( {
                            'Realnumber' => bless( {
                              'Sign' => '',
                              'Value' => '1',
                              'trailingws' => ''
                            }, 'SpeL::Object::Realnumber' )
                          }, 'SpeL::Object::Number' )
                        }, 'SpeL::Object::Fraction' ),
                        'Rest' => bless( {
                          'Op' => bless( {
                            'Op' => ''
                          }, 'SpeL::Object::Binop' ),
                          'Remainder' => bless( {
                            'Component' => bless( {
                              'Name' => 'jz',
                              'trailingws' => ' '
                            }, 'SpeL::Object::Command' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Expressionrest' )
                      }, 'SpeL::Object::Expression' )
                    }, 'SpeL::Object::Expressionrest' ),
                    'Sign' => '-'
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Args' => [
                        bless( {
                          '' => 'z',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'z',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' )
                      ],
                      'Name' => 'Ree',
                      'trailingws' => ' '
                    }, 'SpeL::Object::Command' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Sqrt' => bless( {
                        'Argument' => bless( {
                          'Component' => bless( {
                            'Realnumber' => bless( {
                              'Sign' => '',
                              'Value' => '2',
                              'trailingws' => ''
                            }, 'SpeL::Object::Realnumber' )
                          }, 'SpeL::Object::Number' ),
                          'Rest' => bless( {
                            'ws' => ''
                          }, 'SpeL::Object::Expressionrest' )
                        }, 'SpeL::Object::Expression' )
                      }, 'SpeL::Object::Squareroot' )
                    }, 'SpeL::Object::Function' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' ),
                    'Sign' => '-'
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Args' => [
                        bless( {
                          '' => 'z',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'z',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' )
                      ],
                      'Name' => 'Ima',
                      'trailingws' => ' '
                    }, 'SpeL::Object::Command' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'den' => bless( {
                        'Realnumber' => bless( {
                          'Sign' => '',
                          'Value' => '2',
                          'trailingws' => ''
                        }, 'SpeL::Object::Realnumber' )
                      }, 'SpeL::Object::Number' ),
                      'num' => bless( {
                        'Realnumber' => bless( {
                          'Sign' => '',
                          'Value' => '1',
                          'trailingws' => ''
                        }, 'SpeL::Object::Realnumber' )
                      }, 'SpeL::Object::Number' )
                    }, 'SpeL::Object::Fraction' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' ),
                    'Sign' => '-'
                  }, 'SpeL::Object::Expression' )
                }
              ]
            }, 'SpeL::Object::Relation' ),
            bless( {
              '' => '(c) && z=-\\pi \\jz && \\Ree{z} = 0 && \\Ima{z} = - \\pi
',
              'Left' => bless( {
                'Component' => bless( {
                  '' => 'c',
                  'Left' => bless( {
                    'Component' => bless( {
                      'Alphabet' => 'c',
                      'trailingws' => ''
                    }, 'SpeL::Object::Variable' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                }, 'SpeL::Object::Relation' ),
                'Rest' => bless( {
                  'ws' => ''
                }, 'SpeL::Object::Expressionrest' ),
                'cbracket' => ')',
                'obracket' => '('
              }, 'SpeL::Object::Expression' ),
              'RelationChain' => [
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Alphabet' => 'z',
                      'trailingws' => ''
                    }, 'SpeL::Object::Variable' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Realnumber' => bless( {
                        'Sign' => '-',
                        'Value' => '\\pi',
                        'trailingws' => ' '
                      }, 'SpeL::Object::Realnumber' )
                    }, 'SpeL::Object::Number' ),
                    'Rest' => bless( {
                      'Op' => bless( {
                        'Op' => ''
                      }, 'SpeL::Object::Binop' ),
                      'Remainder' => bless( {
                        'Component' => bless( {
                          'Name' => 'jz',
                          'trailingws' => ' '
                        }, 'SpeL::Object::Command' ),
                        'Rest' => bless( {
                          'ws' => ''
                        }, 'SpeL::Object::Expressionrest' )
                      }, 'SpeL::Object::Expression' )
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Args' => [
                        bless( {
                          '' => 'z',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'z',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' )
                      ],
                      'Name' => 'Ree',
                      'trailingws' => ' '
                    }, 'SpeL::Object::Command' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Realnumber' => bless( {
                        'Sign' => '',
                        'Value' => '0',
                        'trailingws' => ' '
                      }, 'SpeL::Object::Realnumber' )
                    }, 'SpeL::Object::Number' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '&'
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Args' => [
                        bless( {
                          '' => 'z',
                          'Left' => bless( {
                            'Component' => bless( {
                              'Alphabet' => 'z',
                              'trailingws' => ''
                            }, 'SpeL::Object::Variable' ),
                            'Rest' => bless( {
                              'ws' => ''
                            }, 'SpeL::Object::Expressionrest' )
                          }, 'SpeL::Object::Expression' )
                        }, 'SpeL::Object::Relation' )
                      ],
                      'Name' => 'Ima',
                      'trailingws' => ' '
                    }, 'SpeL::Object::Command' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' )
                  }, 'SpeL::Object::Expression' )
                },
                {
                  'RelOperator' => bless( {
                    '' => '='
                  }, 'SpeL::Object::RelOperator' ),
                  'Right' => bless( {
                    'Component' => bless( {
                      'Realnumber' => bless( {
                        'Sign' => '',
                        'Value' => '\\pi',
                        'trailingws' => '
'
                      }, 'SpeL::Object::Realnumber' )
                    }, 'SpeL::Object::Number' ),
                    'Rest' => bless( {
                      'ws' => ''
                    }, 'SpeL::Object::Expressionrest' ),
                    'Sign' => '-'
                  }, 'SpeL::Object::Expression' )
                }
              ]
            }, 'SpeL::Object::Relation' )
          ],
          'mathtag' => 'align',
          'trailingws' => '
  '
        }, 'SpeL::Object::MathEnvironment' ),
        bless( {
          '' => '
'
        }, 'SpeL::Object::TokenSequence' )
      ]
    }, 'SpeL::Object::ElementList' )
  }, 'SpeL::Object::Document' )
}, 'SpeL::Parser::Chunk' );

